sound/soc/intel/common/soc-acpi-intel-lnl-match.c

```diff
@@ -117,6 +117,27 @@ static const struct snd_soc_acpi_endpoint cs42l43_endpoints[] = {
 	},
 };
 
+static const struct snd_soc_acpi_endpoint cs42l43_endpoints_nomic[] = {
+	{ /* Jack Playback Endpoint */
+		.num = 0,
+		.aggregated = 0,
+		.group_position = 0,
+		.group_id = 0,
+	},
+	{ /* Jack Capture Endpoint */
+		.num = 2,
+		.aggregated = 0,
+		.group_position = 0,
+		.group_id = 0,
+	},
+	{ /* Speaker Playback Endpoint */
+		.num = 3,
+		.aggregated = 0,
+		.group_position = 0,
+		.group_id = 0,
+	},
+};
```

Comment:
Or use a dai quick. Like https://github.com/thesofproject/linux/pull/4848/commits/47bae326da237ca6cf0414dc5a8a823d1ff83a39

Reply:
Yeah I did kinda want to use a quirk, but it was a little awkward, since the polarity of the quirks is such that they are saying include this DAI. Which means we would need to add the quirk to every laptop except this one and given generally I expect the vast majority of SKUs to use the cs42l43 DMICs that feels suboptimal. I will have another look today see if I can work out a better way to invert the quirk polarity.